lib/screens/statistiken/authenticate_screen.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
```

Comment:
Falscher Ordner. Ggf /utility_screens anlegen und auch den Loading Screen rein legen

Reply:
Login Screen könnte da auch noch rein. Ich würde eher `/screens/utilities` vorschlagen

Reply:
Ich habe jetzt loading und authentication screen in `/screens/utilities` gepackt.